go.mod

```
@@ -98,8 +98,10 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
 
-// replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 dev
+replace github.com/jfrog/jfrog-cli-core/v2 => github.com/gailazar300/jfrog-cli-core/v2 v2.0.0-20240403103644-317f145c1f66
 
-// replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go dev
+//replace github.com/jfrog/jfrog-cli-core/v2 => ../jfrog-cli-core
```

```suggestion
replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 dev

replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go dev
```
Update to dev branch, those changes already merged

---

again- of course, I wont merge the go.mod like this!
and will run tests with the dev branches, no need to write it every time.